content/discoverable/fix-http-status-codes/index.md

```diff
@@ -1,7 +1,12 @@
 ---
 page_type: guide
 title: Fix unsuccessful HTTP status codes
-author: ekharvey
+author:
```

Comment:
Need to revert this, but it gives you an impression of what information is required.